src/parser/shared/modules/items/bfa/AncientKnotOfWisdom.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+
+import SPELLS from 'common/SPELLS/index';
+import ITEMS from 'common/ITEMS/index';
+import Analyzer from 'parser/core/Analyzer';
+import Abilities from 'parser/core/modules/Abilities';
+import StatTracker from 'parser/shared/modules/StatTracker';
+import { formatPercentage } from 'common/format';
+import { formatNumber } from 'common/format';
+import { calculateSecondaryStatDefault } from 'common/stats';
+
+/*
+* Ancient Knot of Wisdom
+* Heroic
+* Item Level 385
+* Binds when picked up
+* Unique-Equipped
+* Trinket
+* +176 Mastery
+* Use: Increase your Intellect by 1060 decreasing by 212 every 4 sec. (1 Min Cooldown)
+*
+* Testing Log:
+* https://www.warcraftlogs.com/reports/t2ZK6pQvgLGqj7by#fight=1&source=13
+*/
+
+class AncientKnotOfWisdom extends Analyzer {
+  static dependencies = {
+    abilities: Abilities,
+    statTracker: StatTracker,
+  };
+  
+  casts = 0;
+  intellect = 0;
+
+  constructor(...args) {
+    super(...args);
+    this.active = this.selectedCombatant.hasTrinket(
+      ITEMS.ANCIENT_KNOT_OF_WISDOM.id
+    );
+    if (this.active) {
+      this.intellect = calculateSecondaryStatDefault(385, 1060, this.selectedCombatant.getItem(ITEMS.ANCIENT_KNOT_OF_WISDOM.id).itemLevel);
```

Comment:
After lengthy discussion on Discord we figured out the most correct solution is to use per-stack intellect values, and scaling from the base ilvl of 355 (as is on Wowhead), which is 800 intellect (40 intellect per stack).
```suggestion
      this.intellectPerStack = calculatePrimaryStat(355, 40, this.selectedCombatant.getItem(ITEMS.ANCIENT_KNOT_OF_WISDOM.id).itemLevel);
```
This will return intellect per stack for the given item level, which you can directly plug in the statTracker line and also use that for the average intellect.

Note: scaling this off 385 version (53 intellect per stack) isn't necessarily wrong, but because of rounding errors, this ends up being 1 intellect per stack off, resulting in 20 intellect difference, that's why I suggested base ilvl